packages/mobile/src/stableToken/saga.test.ts

```diff
@@ -93,16 +108,6 @@ describe('stableToken saga', () => {
       .run()
   })
 
-  // TODO(cmcewen): Figure out how to mock this so we can get actual contract calls
```

Comment:
Note that I removed this test as a `getStableToken` call is already tested in the 'should fetch the balance and put the new balance' test above.